### PR TITLE
[MRG] Improve Python intro

### PR DESCRIPTION
**Pull request by cbrnr**
Fix some typos and use clearer language.

Any ideas how to include that long NumPy for MATLAB users link without rendering the `# noqa` tag in the HTML page?

### CODE REVIEW

**Comment by agramfort** [comment]
thanks for taking a stab at this.

please put your PR to MRG when you're done

**Comment by cbrnr** [comment]
Here's the output:
https://6290-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.8wJCjs3/html/auto_tutorials/plot_python_intro.html

Looks good except for the `# noqa` after the link. Do we pep-check tutorials? How do you handle long links within 79 chars?

File: tutorials/plot_python_intro.py [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
 
 * `SciPy lectures <http://scipy-lectures.github.io>`_
 * `Learn X in Y minutes: Python <https://learnxinyminutes.com/docs/python/>`_
 * `NumPy for MATLAB users <https://docs.scipy.org/doc/numpy-dev/user/numpy-for-matlab-users.html>`_  # noqa
 
-We highly recommend watching the Scipy videos and reading through these
-sites to get a sense of how scientific computing is done in Python.
+We highly recommend watching the SciPy videos and reading through these sites
+to get a sense of how scientific computing is done in Python.
 """
```
Comment (agramfort) [inline tutorials/plot_python_intro.py]: # noqa

should be here at end of docstring not inside it.

**Comment by cbrnr** [comment]
Looks good now: https://6291-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.SYoDgML/html/auto_tutorials/plot_python_intro.html

**Comment by agramfort** [comment]
thx @cbrnr

**Pull request merged**